src/main/java/seedu/cafectrl/command/ListMenuCommand.java

```diff
@@ -40,7 +40,7 @@ public void printFullMenu(Menu menu, Ui ui) {
             String indexNum = String.valueOf(i+1);
```

Comment:
Perhaps add whitespace in between operators
```suggestion
            String indexNum = String.valueOf(i + 1);
```